src/codewhisperer/commands/basicCommands.ts

```diff
@@ -64,16 +73,27 @@ export const requestAccess = Commands.declare('aws.codeWhisperer.requestAccess',
 export const requestAccessCloud9 = Commands.declare(
     'aws.codeWhisperer.requestAccessCloud9',
     (context: ExtContext) => async () => {
```

Comment:
Looks like we can make it even simpler which means simpler tests too:
```suggestion
    (globalState: vscode.Memento) => async () => {
```

Then change `context.extensionContext.globalState` -> `globalState` 
Same applies to the other command